sql/flow_config.go

```diff
@@ -18,6 +18,15 @@ type configResponse struct {
 	BaseDockerImage string `json:"baseDockerImage"`
 }
 
+type compatibilityVersions struct {
+	AstroRuntime   string `json:"astro-runtime"`
+	AstroSdkPython string `json:"astro-sdk-python"`
```

Comment:
@pankajkoti what do you think if we check both for the lower and upper bound versions?
https://github.com/astronomer/astro-sdk/issues/1473

```
            "astroSDKPythonMaxVersion": "",
            "astroSDKPythonMinVersion": "1.3.1",
            "astroRuntimeMinVersion": "6.1.0",
            "astroRuntimeMaxVersion": "7.1.0"
```

This would be helpful if we need to introduce breaking changes in the future.

Reply:
Maybe we can have sth. like `>6.1.0,<7.1.0` syntax as in python. Maybe go-version supports that?

Reply:
Handling as per Felix's suggestions.